modules/tinymce/src/themes/silver/main/ts/ui/image/Images.ts

```diff
@@ -0,0 +1,84 @@
+import { AddEventsBehaviour, AlloyEvents, Behaviour, SimpleSpec } from '@ephox/alloy';
+import { Arr, Obj } from '@ephox/katamari';
+import { Css, Insert, Remove, SelectorFind, SugarElement } from '@ephox/sugar';
+import createDompurify from 'dompurify';
+
+export type ImageProvider = () => Record<string, string>;
+
+interface ImageSpec {
+  readonly tag: string;
+  readonly classes: string[];
+  readonly attributes?: Record<string, string>;
+  readonly behaviours?: Array<Behaviour.NamedConfiguredBehaviour<any, any, any>>;
+}
+
+const getInnerHTML = (url: string): string => createDompurify().sanitize(`
+  <div style="width: 46px; height: 46px; display: flex; align-items: center; justify-content: center;">
```

Comment:
This needs to be moved to CSS.

- [ ] Move hardcoded styles to CSS

Reply:
I think this code doesn't exists anymore